course_catalog/etl/youtube_test.py

```diff
@@ -100,36 +108,21 @@ def mock_channel_data():
 
 
 @pytest.fixture
-def mocked_channel_response(mocked_responses, mock_channel_data, settings):
+@pytest.mark.usefixtures("video_settings")
+def mocked_channel_response(mocked_responses, mock_channel_data, video_settings):
     """Mock the channel catalog response"""
-    settings.VIDEOS_CONFIG_FILE_LOCATION = "https://youtube.yaml"
-
     mocked_responses.add(
         mocked_responses.GET,
-        settings.VIDEOS_CONFIG_FILE_LOCATION,
+        video_settings.YOUTUBE_CONFIG_FILE_LOCATION,
         body=mock_channel_data,
     )
     yield mocked_responses
 
 
-@pytest.fixture
-def mocked_channel_empty_response(mocked_responses, settings):
-    """Mock the channel catalog response when the response is empty"""
-    settings.VIDEOS_CONFIG_FILE_LOCATION = "https://youtube_empty.yaml"
-
-    mocked_responses.add(
-        mocked_responses.GET, settings.VIDEOS_CONFIG_FILE_LOCATION, body=""
-    )
-    yield mocked_responses
-
-
 @pytest.mark.usefixtures("mocked_channel_response")
-def test_extract(mocker, mock_video_data, mock_video_batch_data, settings):
+@pytest.mark.usefixtures("video_settings")
```

Comment:
Since this is an `autouse=True` fixture, you don't need `usefixtures` for it here.